packages/manager/modules/pci/src/projects/project/project.html

```diff
@@ -58,6 +58,74 @@
                                     data-translate-values="{ quotaUrl: $ctrl.getQuotaUrl() }"
                                 ></p>
                             </oui-message>
+                            <!-- mobile display -->
+                            <oui-message
+                                class="d-block d-sm-none"
+                                data-type="info"
+                            >
+                                <div
+                                    class="d-flex flex-row justify-content-between align-items-center"
+                                >
+                                    <div class="mr-3 mb-3">
+                                        <h6 class="m-0">
+                                            <span
+                                                data-translate="pci_projects_project_discovery_banner_title_1"
+                                            ></span>
+                                            <span
+                                                data-translate="pci_projects_project_discovery_banner_title_2"
+                                            >
+                                            </span>
+                                        </h6>
+                                        <span
+                                            data-translate="pci_projects_project_discovery_banner_description"
+                                        ></span>
+                                    </div>
+                                </div>
+                                <oui-button
+                                    class="d-block d-sm-none"
+                                    size="l"
+                                    block
+                                    variant="primary"
+                                    on-click="$ctrl.goToDiscoveryProjectActivationPage()"
+                                >
+                                    <span
+                                        data-translate="pci_projects_project_discovery_banner_cta"
+                                    ></span>
+                                </oui-button>
+                            </oui-message>
+                            <!-- Tablet & desktop display -->
+                            <oui-message
+                                class="d-none d-sm-block"
```

Comment:
**question:** with `d-none` when will this element be displayed?

Reply:
This element should appear only for the mobile and hidden for the other displays.
https://getbootstrap.com/docs/4.0/utilities/display/
I see however ,that the comment is not correctly placed. 
So I've updated my PR to have the correct comment.